### ISSUE TITLE
depth map of image_c

### ISSUE THREAD

**Issue by zzq123zz** [simonwu53/NetCalib2-Sensors-Auto-Calibration#5]
Can image_c be obtained directly from the data_depth_annotated without having to generate it yourself?

**Comment by simonwu53** [comment]
Yes, if you have a depth camera or stereo camera, you can use the depth images directly. Noted that the LiDAR projection map should have the same size as depth maps.

**Comment by zzq123zz** [comment]
Didn't your paper say that the exact depth map obtained from the KITTI depth dataset refers to data_depth_annotated?How you get the image depth map when you implement it？
















At 2023-10-11 17:14:44, "吴杉(Shan Wu)" ***@***.***> wrote:

Yes, if you have a depth camera or stereo camera, you can use the depth images directly. Noted that the LiDAR projection map should have the same size as depth maps.

**Comment by simonwu53** [comment]
Yes, if you just use KITTI, you can obtain depth maps by OpenCV block matching algorithm. For example, [this link](https://learnopencv.com/depth-perception-using-stereo-camera-python-c/) implements it or you can do it yourself using [OpenCV official tutorial](https://github.com/PacktPublishing/OpenCV-with-Python-By-Example/blob/master/Chapter11/stereo_match.py).
